src/Workspaces/CSharpTest/CodeGeneration/SyntaxGeneratorTests.cs

```diff
@@ -1001,6 +1001,24 @@ public void TestConversionOperatorDeclaration()
                 "public static implicit operator global::System.Decimal(global::System.Byte value)\r\n{\r\n}");
         }
 
+        [Fact, WorkItem(77101, "https://github.com/dotnet/roslyn/issues/77101")]
+        public void TestExplicitOperatorFromSymbol()
+        {
+            var compilation = CSharpCompilation.Create("Test", [SyntaxFactory.ParseSyntaxTree("""
+                public class C
+                {
+                    public static explicit operator int(C c) => 0;
```

Comment:
When adding a `checked` explicit operator, a Roslyn analyzer warns you that you also need to add an unchecked one:
```suggestion
                    public static explicit operator int(C c) => (int)(C)value;
                    public static explicit operator checked int(C c) => checked((int)(C)value);
```